neet/information.py

```diff
@@ -0,0 +1,350 @@
+# Copyright 2017 ELIFE. All rights reserved.
+# Use of this source code is governed by a MIT
+# license that can be found in the LICENSE file.
+import numpy as np
+import pyinform as pi
+from neet.synchronous import timeseries
+
+def active_information(net, k, timesteps, size=None, local=False):
+    """
+    Compute the active information storage for each node in a network.
+
+    ::
+
+        >>> active_information(s_pombe, k=5, timesteps=20)
+        array([ 0.        ,  0.4083436 ,  0.62956679,  0.62956679,  0.37915718,
+                0.40046165,  0.67019615,  0.67019615,  0.39189127])
+        >>> lais = active_information(s_pombe, k=5, timesteps=20, local=True)
+        >>> lais[1]
+        array([[ 0.13079175,  0.13079175,  0.13079175, ...,  0.13079175,
+                0.13079175,  0.13079175],
+            [ 0.13079175,  0.13079175,  0.13079175, ...,  0.13079175,
+                0.13079175,  0.13079175],
+            [ 0.13079175,  0.13079175,  0.13079175, ...,  0.13079175,
+                0.13079175,  0.13079175],
+            ...,
+            [ 0.13079175,  0.13079175,  0.13079175, ...,  0.13079175,
+                0.13079175,  0.13079175],
+            [ 0.13079175,  0.13079175,  0.13079175, ...,  0.13079175,
+                0.13079175,  0.13079175],
+            [ 0.13079175,  0.13079175,  0.13079175, ...,  0.13079175,
+                0.13079175,  0.13079175]])
+        >>> np.mean(lais[1])
+        0.40834359639631324
+
+    :param net: a NEET network
+    :param k: the history length
+    :param timesteps: the number of timesteps to evaluate the network
+    :param size: the size of variable-sized network (or ``None``)
```

Comment:
Can we maybe change the :param size: description to: "the size of the network (only needs to be specified when network is not fixed size, e.g. an ECA)" or something like that? I just think it's a little vague the way it's written. This would apply to any functions which take this argument.